modules/rosa-install-logging-addon.adoc

```diff
@@ -0,0 +1,76 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-install-logging-addon_{context}"]
+= Install logging add-on services on {product-title}
+
+{product-tite} (ROSA) does not come preconfigured with a logging solution, but logging can be enabled by manually installing the `cluster-logging-operator` service.
+
+Log forwarding capability is provided through the Amazon CloudWatch add-on service.
+
+.Prerequisites
+
+Request the following link:https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html[quotas from AWS]:
+* Quota for the `cluster-logging-operator` add-on
+* Quota for the Amazon CloudWatch add-on
+
+.Procedure
+
+To install log forwarding using interactive mode:
+
+. Enter the following command:
++
+[source,terminal]
+----
+$ rosa install addon cluster-logging-operator --cluster=<cluster_name> --interactive
+----
++
+For `<cluster_name>`, enter the name of your cluster.
+
+. When prompted, enter `yes` to install `cluster-logging-operator`.
+. When prompted, enter `yes` to install the optional Amazon CloudWatch log forwarding add-on.
+. Enter `yes` or `no` for the log type prompts: applications, infrastructure, and audit logs.
++
+*Applications logs*
++
+Lets the Operator collect application logs, which includes everything that is *not* deployed in the `openshift-*` namespaces.
+
++
+*Infrastructure logs*
++
+Lets the Operator collect logs from OpenShift Container Platform, Kubernetes, and some nodes.
+
++
+*Audit logs*
++
+Lets the Operator collect node logs related to security audits.
+
+. If you are installing the Amazon CloudWatch add-on, enter the log grouping type: `logType`, `<namespaceName>``, or `<namespaceUUID>`.
+. Optional: Enter the Amazon CloudWatch region. To use the default cluster region, leave the `Cloudwatch region` value empty.
+
++
+.Example output
+[source,terminal]
+----
+? Are you sure you want to install add-on 'cluster-logging-operator' on cluster '<cluster_name>'? Yes
+? Use AWS CloudWatch (optional): Yes
+? Collect Applications logs (optional): Yes
+? Collect Infrastructure logs (optional): No
+? Collect Audit logs (optional): Yes
+? CloudWatch log grouping type: logType
+? CloudWatch region (optional):
+I: Add-on 'cluster-logging-operator' is now installing. To check the status run 'rosa list addons --cluster=<cluster_name>'
+----
+
+[NOTE]
```

Comment:
Just checking that this NOTE isn't supposed to be indented with step 6. If it is, it's missing the + on the line above it.

Reply:
I didn't intend to indent it because it's a general note for the procedure. I hope that doesn't mess up the validation step numbering. I'll have to check it when I can publish a preview, after this is merged. We know there may some cleanup.